packages/panel/src/hooks/useCalendar.ts

```diff
@@ -0,0 +1,82 @@
+import { useAsync } from 'react-use';
+import { ISchedule } from 'tui-calendar';
+
+import { colors } from '@coderscamp/ui/theme/overwrites/foundations/colors';
```

Comment:
🤔 We shouldn't import colors directly as it could be a blocker for future improvements (e.g. dark mode). It's better to re-export use the `useTheme` hook.